mapping/process-level.csv

```diff
@@ -0,0 +1,20 @@
+CoST IDS Element,Draft definition,Maps to,Discussion
+Procuring entity,Enter name of the organisation carrying out the procurement,Project Level: Add an entry to ``parties`` with 'procuringEntity' included in its ``.roles`` | Contracting process: Record the name and identifier in ``contractingProcesses.procuringEntity``,
+Procuring entity contact details,Postal and Electronic address,Project Level: Add an entry to ``parties`` with 'procuringEntity' included in its ``.roles`` | Contracting process: record the name and identifier in `contractingProcesses.procuringEntity`,
+Contract administrative entity,Enter name of the organisation carrying out the contract administrative entity if different from the Procuring Entity,Project Level: Add an entry to ``parties`` with 'administativeEntity' included in its ``.roles`` | Contracting process: Record the name and identifier in `contractingProcesses.administrativeEntity`,

```

Comment:
Typo: Need double-backticks, as I assume this is interpreted as RST and not Markdown.

Reply:
Addressed in https://github.com/open-contracting/infrastructure/commit/66fa0b19a3c440bb1f2b175c88a0c1666f062486